tools/ci/scripts/macos/package_tests.sh

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env bash
```

Comment:
What's this file for?

Reply:
That comes from this commit https://github.com/osquery/osquery/pull/7330/commits/7c76b89c448832f4c8bd2bed0dd5cb1fff50f623
It's meant to be used to take the tests that have been built on the macOS 11 worker and have them run on the 10.15 worker, without rebuilding them.
We wanted to keep testing both the newer and older version for compatibility.

Reply:
That was my guess. Not sure it's worth a comment in the script or not. I'm loathe to make any changes to this PR.